### PR TITLE
Donations

### PR DESCRIPTION
**Pull request by jaywayawyaj**
Check this out! Definitely works through the API if hosted locally - Heroku, not so much :)

### CODE REVIEW

**Comment by jaywayawyaj** [comment]
Also It's right big, sorry

File: src/components/donation.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+  handleForm = (event) => {
+    var amount = document.getElementById('amount').value;
+    var passphrase = document.getElementById('passphrase').value;
+    this.sendDonation(amount, passphrase)
+    //this.updateState(amount, passphrase)
+  }
+
+  sendDonation = (amount, passphrase) => {
+    // let venue = this.state.venue.id;
```
Comment (stonefarmer9) [inline src/components/donation.js:21]: Can these comments be removed?

File: src/components/donation.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+          />
+        </div>
+        :
+        <h1>Thanks for your donation</h1>}
+      </div>
+    )
+  }
+}
+
+// https://enigmatic-badlands-83570.herokuapp.com/api/v1/venues/${venue}/donations
```
Comment (stonefarmer9) [inline src/components/donation.js:61]: Do we need this comment?

**Review by joaoag — APPROVED** [review]
Looks good, I'm happy to merge.

File: src/test/App.test.js [hunk 3]
```diff
@@ -3,21 +3,46 @@ import React, {Component} from 'react';
...
+    })
 
     it('should say hello', ()=>{
         expect(
             wrapper.containsMatchingElement(
-                <h1>Hello, React!</h1>
+                <h1>Keep Ahead</h1>
             )
         ).toBeTruthy()
     })
```
Comment (stonefarmer9) [inline src/test/App.test.js:27]: We had issues with toBeTruthy comparing elements, as a variable will be truthy if it has value, did this issue come up for you and if so how did you combat it?

File: src/test/features/mvpFeature.test.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import App from '../../App'
+import React, {Component} from 'react';
+import {shallow, mount, render} from 'enzyme'
+
+
+describe('MVP feature', () => {
+  xit('You can choose a venue from list, see details and donate', () => {
```
Comment (stonefarmer9) [inline src/test/features/mvpFeature.test.js:7]: do we need to send this test to alice still if it is commented out? Or do we need to re-write it or re-enable it?

**Review by stonefarmer9 — APPROVED** [review]
Approved but we need to remove some of these comments I think?

**Pull request merged**